source/reflect.h

```diff
@@ -3342,22 +3343,22 @@ std::string str {s}; return from_string(cpp2::string_util::replace_all(cpp2::mov
 
     regex_token_list::~regex_token_list() noexcept{}
 
-#line 1672 "reflect.h2"
+#line 1673 "reflect.h2"
     auto parse_context_group_state::next_alternative() & -> void{
         token_vec new_list {}; 
         std::swap(new_list, cur_match_list);
         post_process_list(new_list);
         static_cast<void>(alternate_match_lists.insert(alternate_match_lists.end(), CPP2_UFCS_TEMPLATE(cpp2_new<regex_token_list>)(cpp2::shared, cpp2::move(new_list))));
     }
 
-#line 1680 "reflect.h2"
-    auto parse_context_group_state::swap(parse_context_group_state& t) & -> void{
+#line 1681 "reflect.h2"
+    auto parse_context_group_state::swap(parse_context_group_state& t) & -> void{// NOLINT(performance-noexcept-swap)
```

Comment:
Can this method be marked `noexcept` instead?

Reply:
@jarzec probably better to review the `.h2` instead of the `.h`.

Reply:
the stuff in reflect.h is generated code, please provide alternative syntax for reflect.h2 and I will update it!

Reply:
If this is generated shouldn't it be excluded from git?

Reply:
No, because otherwise `cppfront` isn't self-hosting, you need a copy of `cppfront` to build `cppfront`.